plugins/lang-plan-a/src/main/java/org/elasticsearch/plan/a/PlanAParserBaseVisitor.java

```diff
@@ -0,0 +1,357 @@
+// ANTLR GENERATED CODE: DO NOT EDIT
+package org.elasticsearch.plan.a;
```

Comment:
Then maybe hipchat or an email list is a more suitable place. In this broken dev culture, every single comment/question must be addressed, so it comes across as laziness.